components/BaseLinePlotComponent.vue

```diff
@@ -234,16 +238,46 @@ export default class BaseLinePlotComponent extends Vue {
     return this.cumulatedSplitTimes[virtualCurrentSplitIndex] || [];
   }
 
+  get goldDate() {
+    const attempts = this.parsedSplits.Run.AttemptHistory.Attempt;
+    const dateTime = attempts.find(a => this.timesToPlot[this.gold.x]['@_id'] === a['@_id'])?.['@_ended'].split(" ");
```

Comment:
Wouldn't `@_started` make more sense than `@_ended` ? (not just here, everywhere) It's not very important, but I think it makes more sense to compare the start of each run, because the end will be more or less late depending on the duration of the attempt (if you finish or not, or where you reset). So the start is a more "consistent" measurement of when a run happens (again it doesn't matter too much, it's a very small detail)